app/homepage.py

```diff
@@ -389,22 +328,23 @@ def Homepage():
                 id='login_background',
                 children=[
                     # The background image is directly stored in the /assets folder
-                    html.Img(src='assets/background.png', width="100%")
+                    html.Img(src='assets/background.png', width="100%", height="100%")
```

Comment:
Regarding the issue that you pointed out (the fact that we can see the map even when not yet logged in, behind the modal), I will push a workaround in my next PR. The style attribute of the "hp_map" component is simply updated with `{'display': 'none'}` as long as the user is not logged in and with `{}` once login is successful, letting the map appear.

Reply:
Alright, great !